### PR TITLE
[bitnami/tomcat] add containerExtraPorts

### PR DESCRIPTION
**Pull request by itscaro**
**Benefits**
- Add the possibility to define extra ports listened by tomcat

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/tomcat/README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ The following tables lists the configurable parameters of the Tomcat chart and t
 |-----------------------------|-------------------------------------------------------------------------------------------|---------------------------------------------|
 | `replicaCount`              | Specify number of Tomcat replicas                                                         | `1`                                         |
 | `containerPort`             | HTTP port to expose at container level                                                    | `8080`                                      |
+| `containerExtraPorts`       | Extra HTTP ports to expose at container level                                                    | `{}`                                      |
```
Comment (juan131) [inline bitnami/tomcat/README.md:98]: The ports to open can potentially use any protocol. Therefore, I'd remove the "HTTP" reference:
```suggestion
| `containerExtraPorts`       | Extra ports to expose at container level                                               | `{}`                                      |
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for this great contribution! I just added a minor request.

File: bitnami/tomcat/README.md (hunk 1 above)
Comment (itscaro) [reply bitnami/tomcat/README.md:98]: right I missed this ;) (and in example I used AJP)

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**